### ISSUE TITLE
code in 'chapter4/before' missing?

### ISSUE THREAD

**Issue by benpeart** [chaostoolkit-incubator/community-playground#27]
The book references "This code is available in the chapter4/ before directory in the sample code for this book that you grabbed earlier (see “Setting Up the Sample Target System”)." but I can't find it anywhere.

**Comment by russmiles** [comment]
Taking a look at this now @benpeart

**Comment by russmiles** [comment]
Thanks for raising the issue!

**Comment by russmiles** [comment]
I'm not seeing that reference in my copy @benpeart , not in Chapter 4 anyway. Chapter 5 and 6 are present though, and it's chapter 6 I believe that has the `before` and `after` approach. I was wondering if you had a print copy of the book or perhaps the pre-release/early release copy?

**Comment by benpeart** [comment]
I purchased the Kindle edition on Aug 27th so I don't _think_ I have a pre-release copy.  The text in the issue above is a direct quote that can be found in Chapter 6 (about the 2nd 'page') and is also known as "Location 1171" in the Kindle edition.  I've linked a screen shot of the text.
![Chaos Engineering reference to missing sample code ](https://user-images.githubusercontent.com/15252029/64172636-6ed2d980-ce23-11e9-8b9a-0d1e4040cf6a.jpg)
